### ISSUE TITLE
Blank line between JSDoc and function means JSDoc @params not checked

### ISSUE THREAD

**Issue by yphoenix** [gajus/eslint-plugin-jsdoc#1133]
## Expected behavior
```
/**
 * @param {string} arg42
 */

function foo(arg99)
{
}

  NN:1   error  Missing JSDoc @param "arg99" declaration          jsdoc/require-param
  NN:1   error  Expected @param names to be "arg99". Got "arg42"  jsdoc/check-param-names
  NN:14  error  'arg99' is defined but never used                 no-unused-vars
```

## Actual behavior
```
  44:14  error  'arg99' is defined but never used  no-unused-vars
```

If I remove the blank line between the end of the JSDoc and the function everything works as expected.

## Environment

- Node version: v20.4.0
- ESLint version v8.45.0
- `eslint-plugin-jsdoc` 46.4.4

**Comment by brettz9** [comment]
You can increase settings' `maxLines`:

```js
{
  settings: {
    jsdoc: {
      maxLines: 100
    }
  }
}
```

Closing as that should resolve but feel free to comment further as necessary.
